### ISSUE TITLE
implement_fixed_controls in fit_all_gmm_models

### ISSUE THREAD

**Issue by jadenecke** [ucl-pond/kde_ebm#15]
I am trying to use [pySuStaIn](https://github.com/ucl-pond/pySuStaIn) and the [validation.py](https://github.com/ucl-pond/pySuStaIn/blob/master/tests/validation.py) file there. It uses kde_ebm. When I try to run it, i get the following error:

```
File "/home/jdenecke/.conda/envs/sustain/lib/python3.7/site-packages/kde_ebm/mixture_model/utils.py", line 54, in fit_all_gmm_models
    mm.fit(bio_X, bio_y, implement_fixed_controls=implement_fixed_controls)
TypeError: fit() got an unexpected keyword argument 'implement_fixed_control
```

I think it is because of a bug in this package. In  in [`utils.py`](https://github.com/ucl-pond/kde_ebm/blob/master/kde_ebm/mixture_model/utils.py) you define `fit_all_gmm_models`and within there is the following chunk:

```
mm = ParametricMM(cn_comp, ad_comp)
mm.fit(bio_X, bio_y, implement_fixed_controls=implement_fixed_controls)
```
As far as I can see it refers to `class ParametricMM()` in the [`gmm.py`](https://github.com/ucl-pond/kde_ebm/blob/master/kde_ebm/mixture_model/gmm.py) which has a subroutine called:

```
 def fit(self, X, y):
```

which does not specify the `implement_fixed_controls` parameter and leads to the above stated error. 

I am not fully certain whether this is correctly summarized, but it would be nice if you could have a look.

**Comment by davecash75** [comment]
I had this issue too. I discussed with Neil, and I've proposed a potential fix on a pull request.

**Comment by noxtoby** [comment]
Fixed by [d596810](https://github.com/ucl-pond/kde_ebm/commit/d59681005bf7bf5dade5d4d7c5cabcb69ab53ce4), courtesy of @davecash75
